src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -36,7 +36,6 @@ import {
   ScopedTutorialContextFactory,
 } from './lib/tutorials_registry_types';
 import { tutorialSchema } from './lib/tutorial_schema';
-import { builtInTutorials } from '../../tutorials/register';
```

Comment:
this is an unused import, and also the only usage of tutorials under src/plugins/home/server/tutorials

Reply:
@CMDWillYang Either in this PR, or as a standalone followup, can you also delete this comment: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/0ff9bc57b477b1349564299095f4a00c90e0359d/src/plugins/home/server/services/tutorials/tutorials_registry.ts#L97-L100